### PR TITLE
[terra-action-footer] Add theme-context

### PR DESCRIPTION
**Pull request by pranav300**
### Summary

This pr updates terra-action-footer to use the new theme-context to apply themes.

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: packages/terra-action-footer/src/clinical-lowlight-theme/BlockActionFooter.module.scss [hunk 1]
```diff
@@ -0,0 +1,6 @@
+:local {
+  .clinical-lowlight-theme {
```
Comment (emilyrohrbough) [inline packages/terra-action-footer/src/clinical-lowlight-theme/BlockActionFooter.module.scss:2]: Did we want to back-fill all missing css vars with these changes?

File: packages/terra-action-footer/package.json [hunk 2]
```diff
@@ -23,7 +23,8 @@
   "homepage": "https://github.com/cerner/terra-core#readme",
   "peerDependencies": {
     "react": "^16.8.5",
-    "react-dom": "^16.8.5"
+    "react-dom": "^16.8.5",
+    "terra-theme-context": "^1.0.0"
```
Comment (emilyrohrbough) [inline packages/terra-action-footer/package.json:27]: Shouldn't this be a dependency?

Comment (pranav300) [reply packages/terra-action-footer/package.json]: Updated here https://github.com/cerner/terra-core/pull/2955/commits/6146fb9e5371674f76085d02d1f01acd52f3d3b4

File: packages/terra-action-footer/src/clinical-lowlight-theme/BlockActionFooter.module.scss (hunk 1 above)
Comment (pranav300) [reply packages/terra-action-footer/src/clinical-lowlight-theme/BlockActionFooter.module.scss:2]: Updated here https://github.com/cerner/terra-core/pull/2955/commits/6146fb9e5371674f76085d02d1f01acd52f3d3b4

**Comment by pranav300** [comment]
> What about Standard Action Footer and Centered Action Footer?

Standard and Centered Action Footer are just a wrapper around Block Action Footer Wrapper with no themeable vars. So, even if we provide theme-context to them, it will always be the same and get overridden by blockActionFooter.

**Comment by ShettyAkarsh** [comment]
@pranav300 , add a jest test to check whether the theme classnames are applied properly.

**Pull request merged**